sdk/extensions/Microsoft.Extensions.Azure.WebJobs/api/Microsoft.Extensions.Azure.WebJobs.netstandard2.0.cs

```diff
@@ -0,0 +1,17 @@
+namespace Azure.Extensions.WebJobs
```

Comment:
Just so I understand the thought process, we are going to use `Azure.Extensions.WebJobs` here for this binding but continue to use `Microsoft.Azure.WebJobs.Extensions` for the triggers we've already shipped, as we move them to Track 2?

Reply:
Good point. Didn't put too much thought into this namespace. `Azure.Extensions.WebJobs` is most definitely the wrong thing.

Reply:
I also don't like the package name `Microsoft.Extensions.Azure.WebJobs` is too close to `Microsoft.Azure.Extensions.WebJobs`.

Reply:
`Microsoft.Azure.WebJobs.Clients`?

Reply:
Looking at the some of the existing triggers, it seems like some of them will put the types for the bindings either right in `Microsoft.Azure.WebJobs` or use something like `Microsoft.Azure.WebJobs.Extensions.<ServiceName>`.

I think I prefer just re-using `Microsoft.Azure.WebJobs` over `Microsoft.Azure.WebJobs.Clients` (since the clients are not web job clients really).  I can't think of a "service name" to use for the longer pattern that actually looks reasonable (both `Microsoft.Azure.WebJobs.Extensions.Azure` or `Microsoft.Azure.WebJobs.Extensions.AzureClients` feel worse to me.

For the package name the other bindings always seem to follow `Microsoft.Azure.WebJobs.Extensions`.  I wonder if `Microsoft.Azure.WebJobs.Extensions.Clients` or `Microsoft.Azure.WebJobs.Extensions.AzureClients`  or `Microsoft.Azure.WebJobs.Extensions.Azure` would be okay as a name there?

Reply:
Let's try `Microsoft.Azure.WebJobs.Extensions.Clients` so we mention azure once in the name.